WalletWasabi.Fluent/Behaviors/PasteButtonFlashBehavior.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Reactive.Concurrency;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls.ApplicationLifetimes;
+using Avalonia.Threading;
+using ReactiveUI;
+using WalletWasabi.Fluent.Controls;
+using WalletWasabi.Userfacing;
+
+namespace WalletWasabi.Fluent.Behaviors
+{
+	public class PasteButtonFlashBehavior : DisposingBehavior<AnimatedButton>
+	{
+		private CancellationTokenSource? _cts;
+
+		public static readonly StyledProperty<string> FlashAnimationProperty =
+			AvaloniaProperty.Register<PasteButtonFlashBehavior, string>(nameof(FlashAnimation));
+
+		public static readonly StyledProperty<bool> IsEnabledProperty =
+			AvaloniaProperty.Register<PasteButtonFlashBehavior, bool>(nameof(IsEnabled));
+
+		public string FlashAnimation
+		{
+			get => GetValue(FlashAnimationProperty);
+			set => SetValue(FlashAnimationProperty, value);
+		}
+
+		public bool IsEnabled
+		{
+			get => GetValue(IsEnabledProperty);
+			set => SetValue(IsEnabledProperty, value);
+		}
+
+		protected override void OnAttached(CompositeDisposable disposables)
+		{
+			if (AssociatedObject is null)
+			{
+				return;
+			}
+
+			Dispatcher.UIThread.Post(async () => await CheckClipboardForValidAddressAsync(), DispatcherPriority.Loaded);
```

Comment:
`OnAttached` is executed before the property `IsEnabled` gets its bound value. To solve this I use `DispatcherPriority.Loaded` priority but I am not sure about this solution. 

@danwalmsley